draft-ietf-tls-esni.md

```diff
@@ -267,17 +267,20 @@ invalid if maximum_name_length is greater than 256.
 
 extensions
 : A list of extensions that the client can take into consideration when
-generating a Client Hello message. The purpose of the field is to provide room
-for additional features in the future. The format is defined in {{RFC8446}};
-Section 4.2. The same interpretation rules apply: extensions MAY appear in any
-order, but there MUST NOT be more than one extension of the same type in the
-extensions block. An extension may be tagged as mandatory by using an extension
-type codepoint with the high order bit set to 1. A client which receives a
-mandatory extension they do not understand must reject the ECHOConfig content.
-
-Clients MUST parse the extension list and check for unsupported
-mandatory extensions. If an unsupported mandatory extension is
-present, clients MUST reject the ECHOConfig value.
+generating a ClientHello message. The purpose of the field is to provide room
+for additional features in the future. See {{config-extensions}} for guidance
```

Comment:
features or protections

Reply:
Isn't protection a feature?

Reply:
functionality?

Reply:
Functionality is a bit more general -- let's go with that!